Signal/src/view controllers/NewGroupViewController.m

```diff
@@ -403,4 +403,10 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
     return NO;
 }
 
+#pragma mark - UIScrollViewDelegate
+
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView {
```

Comment:
Nice!  I'm definitely going to do the bold thing.  But I'm going to wait until all open PRs are merged.